setup.py

```diff
@@ -38,11 +35,6 @@ def run(self):
         _sdist.run(self)
 
 
-class install(_install):
```

Comment:
I'm a late to the party, I know, but removing this essentially undoes #12, was there a particular reason for it being removed or can it be reinstated?